flavors/lineageos/default.nix

```diff
@@ -142,6 +143,8 @@ in mkIf (config.flavor == "lineageos")
   webview.prebuilt.availableByDefault = mkDefault true;
   removedProductPackages = [ "webview" ];
 
+  apps.updater.flavor = "lineageos";
```

Comment:
mkDefault here?

Reply:
I thought about that, but my thinking was that if someone wants to override it, they can just `mkForce` it.
Although I don't really have any good arguments for or against either approach. So if you prefer `mkDefault`, I'll make this `mkDefault`.

Reply:
It's `internal`, so I suppose it's not a big deal either way. I'd still prefer `mkDefault` though.